src/report/ReportRecordProcessing.tsx

```diff
@@ -366,7 +387,7 @@ export const rendererForType: any = {
   },
   array: {
     type: 'string',
-    renderValue: (c) => RenderArray(c.value),
+    renderValue: (c) => RenderArray(c.value, c.transposedTable ? c.transposedTable : false),
```

Comment:
Shouldn't this be just c.transposedTable? The ternary operation sounds redundant

Reply:
Right